lib/remark-plugins/links/index.js

```diff
@@ -0,0 +1,103 @@
+import { visit } from 'unist-util-visit'
```

Comment:
What's the origin and licence of this file?

Reply:
Oops, I missed this one. I added the license header.